mindmeld/stemmers.py

```diff
@@ -0,0 +1,81 @@
+from abc import ABC, abstractmethod
+import nltk
+
+
+class Stemmer(ABC):
+
+    def __init__(self):
+        self._stemmer = None
+
+    def set_stemmer(self):
+        """
+        This function sets the underlying stemmer. It is lazily initialized
+        """
+        raise NotImplemented
```

Comment:
Should this be abstract?

Reply:
Further -- I think this should just be an abstract property. The implementation can decide whether it's lazy.

Reply:
Resolved